front/tests/pages/scenario-page-model.ts

```diff
@@ -145,17 +145,18 @@ class PlaywrightScenarioPage {
     this.getAllowancesModule = page.locator('.operational-studies-allowances');
     this.getAllowancesStandardSettings = page
       .getByTestId('standard-allowance-group')
+      .first()
```

Comment:
The test worked without this for me. In which case does the fail test without this line ?

Reply:
it was to clearly see that we have two boxes, we had some issues about that but yes I can remove it

Reply:
done